keyboards/keebit/coral/config.h

```diff
@@ -0,0 +1,26 @@
+// Copyright 2023 numToStr (@numToStr)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+/*
+ * Feature disable options
+ *  These options are also useful to firmware size reduction.
+ */
+
+/* disable debug print */
+//#define NO_DEBUG
+
+/* disable print */
+//#define NO_PRINT
+
+/* disable action features */
+//#define NO_ACTION_LAYER
+//#define NO_ACTION_TAPPING
+//#define NO_ACTION_ONESHOT
+
+#define RP2040_BOOTLOADER_DOUBLE_TAP_RESET
+#define RP2040_BOOTLOADER_DOUBLE_TAP_RESET_TIMEOUT 200U
```

Comment:
(just a nitpick, so feel free to ignore this)
200ms seems like a pretty short timeout.  This may be hard to hit consistently, and a longer timeout may be a good idea.

Reply:
I was able to hit it consistently, but yeah it can be increased to 220 or 225.

Reply:
I've now increased it to 220.